### PR TITLE
Update feedback.md

### PR DESCRIPTION
**Pull request by Wajeed-msft**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5e968d1eb961fb32afe36228facf2ecadadad56d/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c5067c6e-ea20-4fb2-8227-07888250451a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b9739f7a6a7799ff7fb7997ff1707f707263e4c2/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5f166228-0331-4288-ab85-80394984f618) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b9739f7a6a7799ff7fb7997ff1707f707263e4c2/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d90ce49-ec33-46a0-8810-287e6a840383) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2709355e8c978e24721edcdcbbb060ce0c3012bf/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b3c75611-7f44-4967-b671-93f56e86280f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c65c2f3a91f4153488c71e619376eeacbf1d00ad/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b01a84f5-5b44-4a1e-9e68-edc96cb01129) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0d5ef99a4fcad63da778f262765476bdc1436255/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b244487a-2eae-4d25-9b14-40881b94e670) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b6ec0dcf3dc39ce7f6d2b55a60b79ca65ca3dac2/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4ccf154e-1fcd-43f4-a62b-895b2e62f499) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f8cd78216227f99a945d834645b88915874aa460/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d699ac56-0f3b-4dc0-ab32-f08089effc89) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d5405dd2f96db7ae3f8a1f9292f9dd7b27829631/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aa54e13f-1392-4685-aa17-9c63ac555224) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fa8b8a0e1f9c6e82dd88063ffd52b516cd1e9ca3/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da335f67-3d3a-4a61-8b16-20f3c3612f52) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/506b45271b26c0029fb3097d1ca4493e4877a022/msteams-platform/feedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f36f9e4e-4e66-4d01-8b6c-006949d27c9a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by Wajeed-msft** [comment]
@surbhigupta12, @ashishguptaiitb - could you please help to review this?

File: msteams-platform/feedback.md [hunk 1]
```diff
@@ -6,40 +6,43 @@ ms.topic: reference
...
 ---
 
-# Microsoft Teams developer community channels
+# Support and feedback
 
-Microsoft Teams community of developers use Stack Overflow to connect with other developers to ideate, get clarifications, and submit queries. In addition, developers can also use other contacts or sites, depending on the type of community help required to submit issues, limitations, and general questions. Developers can also get product-specific information.
+## Product support and service issues
 
-## Product community channels
+Please use **Azure Admin Center** or **Microsoft 365 Admin Center** for any **business-critical issues** which need SLA based support.
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:13]:
```suggestion
Use **Azure Admin Center** or **Microsoft 365 Admin Center** for any **business-critical issues** that need SLA-based support.
``

File: msteams-platform/feedback.md [hunk 2]
```diff
@@ -6,40 +6,43 @@ ms.topic: reference
...
-| **General questions about Microsoft Teams** | Submit your questions on :::image type="icon" source="assets/icons/teams-icon.png" border="false":::[Microsoft Teams Community](https://answers.microsoft.com/en-us/msteams/forum).|
+| **Microsoft Teams product issues and failures** | If you have a Premier support contract for :::image type="icon" source="assets/icons/microsoft-icon.png" border="false"::: Microsoft 365 then visit the Microsoft 365 Admin Center and use the Support menu to [Open a service request ](https://admin.microsoft.com/). |
+| **Azure help and support** | If you have a paid Azure subscription then visit the Azure Admin Center to [Open a ticket](https://ms.portal.azure.com/#blade/Microsoft_Azure_Support/HelpAndSupportBlade/newsupportreq). |
+| **General questions about Microsoft Teams** | Submit your questions on :::image type="icon" source="assets/icons/microsoft-icon.png" border="false"::: [Microsoft Teams Community](https://answers.microsoft.com/en-us/msteams/forum).|
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:19]:
```suggestion
| **Microsoft Teams product issues and failures** | If you have a Premier support contract for :::image type="icon" source="assets/icons/microsoft-icon.png" border="false"::: Microsoft 365, visit the Microsoft 365 Admin Center and use the Support menu to [open a service request ](https://admin.microsoft.com/). |
| **Azure help and support** | If you have a paid Azure subscription, visit the Azure Admin Center to [open a ticket](https://ms.portal.azure.com/#blade/Microsoft_Azure_Support/HelpAndSupportBlade/newsupportreq). |
| **General questions about Microsoft Teams** | If you have general questions on Microsoft Teams, submit your questions on :::image type="icon" source="assets/icons/microsoft-icon.png" border="false"::: [Microsoft Teams Community](https://answers.microsoft.com/en-us/msteams/forum).|
```

File: msteams-platform/feedback.md [hunk 3]
```diff
@@ -6,40 +6,43 @@ ms.topic: reference
...
 
-## Community help
+## Developer community help
+Microsoft Teams community of developers use Stack Overflow and Microsoft Q&A to connect with other developers to ideate, get clarifications, and submit queries. In addition, developers can also use other contacts or sites, depending on the type of community help required to submit issues, limitations, and general questions.
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:22]:
```suggestion
Microsoft Teams community of developers use Stack Overflow and Microsoft Q&A to connect with other developers to ideate, get clarifications, and submit queries. In addition, developers can also use other contacts or sites, depending on the type of community help required to raise issues, limitations, and general questions.
```

File: msteams-platform/feedback.md [hunk 4]
```diff
@@ -6,40 +6,43 @@ ms.topic: reference
...
 
-### Community forums
+### Developer community forums
 
-Get answers to your Microsoft product and service questions with responses from other community members.
+Post your questions and help other community members by sharing and responding to questions related to  Microsoft Teams App Development.
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:26]:
```suggestion
Post your questions and help other community members by sharing and responding to Microsoft Teams App Development questions.
```

File: msteams-platform/feedback.md [hunk 5]
```diff
@@ -6,40 +6,43 @@ ms.topic: reference
...
 
 | **Community channel**|   **Contact**  |
 |:---------------------|:---------------|
+|**Community help** | <ul> <li>:::image type="icon" source="assets/icons/stack-overflow-icon.png" border="false"::: [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams): Use the `microsoft-teams` tag to post the questions. Follow the Stack Overflow guidelines such as provide a descriptive title, a concise problem statement, and details to reproduce the issue. Feature requests or broad questions are off-topic. If you are a new user, see [Stack Overflow Help Center](https://stackoverflow.com/help).</li>  <li>:::image type="icon" source="assets/icons/microsoft-icon.png" border="false"::: [Microsoft Q&A](/answers/topics/office-teams-app-dev.html): Use the `office-teams-app-dev` tag to post questions. We love Stack Overflow and provide extensive support to questions posted on Stack Overflow. Stack Overflow has criteria about appropriate questions and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this. </li><ul> |
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:30]:
```suggestion
|**Community help** | <ul> <li>:::image type="icon" source="assets/icons/stack-overflow-icon.png" border="false"::: [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams): Use the `microsoft-teams` tag to post questions. Follow the Stack Overflow guidelines, such as providing a descriptive title, a concise problem statement, and details to reproduce the issue. Feature requests or broad questions are off-topic. If you are a new user, see [Stack Overflow Help Center](https://stackoverflow.com/help).</li>  <li>:::image type="icon" source="assets/icons/microsoft-icon.png" border="false"::: [Microsoft Q&A](/answers/topics/office-teams-app-dev.html): We love Stack Overflow and provide extensive support to questions posted on Stack Overflow. Use the `office-teams-app-dev` tag to post questions. Stack Overflow has criteria about appropriate questions and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this. </li><ul> |
```

File: msteams-platform/feedback.md [hunk 6]
```diff
@@ -49,15 +52,14 @@ Submit documentation feedback or updates using the community help channels.
...
-| **Bot and Messaging Extension SDK issues or suggestions** | Submit issues and feature requests to the SDK repository for your bot's language ([C#](https://github.com/Microsoft/botbuilder-dotnet/), [JavaScript](https://github.com/Microsoft/botbuilder-js), or [Python](https://github.com/Microsoft/botbuilder-python)). Post How-to questions at [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework%20microsoft-teams) using the `botframework & microsoft-teams` tag. |
-| **Tab SDK issues or suggestions** | Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). Use the `microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams). |
+| **General questions** | Send general questions about the Microsoft Teams developer platform to [Microsoft Teams Community Help](mailto:microsoftteamsdev@microsoft.com). We encourage posting questions on the channels mentioned here and use email only if no other mode of communication is applicable. |
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:62]:
```suggestion
| **General questions** | Send general questions about the Microsoft Teams developer platform to [Microsoft Teams Community Help](mailto:microsoftteamsdev@microsoft.com). We encourage posting questions on the channels mentioned here and using email only if no other mode of communication is applicable. |
```

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Reviewed, please verify

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ec153912e2e93235a96f770aa0df275b265d3304/msteams-platform/feedback.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/644a4384-ae5c-4094-bc03-07072228d098) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ec153912e2e93235a96f770aa0df275b265d3304/msteams-platform/feedback.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5cdd83f4-581a-4fe9-aac3-b81e328b6ecb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9cb76069b664c4cb98ab60315d9738d17a81fbe3/msteams-platform/feedback.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f567e7a9-b357-45c4-97ea-f4e6251ce6ed) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2172e296472fe24953c38c5721a10a9c181d1d5c/msteams-platform/feedback.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7c406575-e7f4-420b-86bc-c25e2fccf548) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2172e296472fe24953c38c5721a10a9c181d1d5c/msteams-platform/feedback.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fa3979d1-1799-4b64-a608-accb7ab36b56) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2172e296472fe24953c38c5721a10a9c181d1d5c/msteams-platform/feedback.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/85fe2e97-a1c5-457f-969b-11c201ad86ad) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2de2e1eb29a7fc85ff169ee584f9203f07faf6a4/msteams-platform/feedback.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/95710405-f25d-4cb4-8ade-168d4250b5b1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fed114b012b04dba544af54212ff273ccb3c58e0/msteams-platform/feedback.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/623d0412-1547-4eff-97ee-66b293f159d7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2aec4ee6160e2679008647575ee82410ddfb6051/msteams-platform/feedback.md) | 92 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/736d50cc-cfe7-4323-a115-72d6c20d0be4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**